src/main/java/com/meetup/server/review/persistence/ReviewRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.meetup.server.review.persistence;
+
+import com.meetup.server.review.domain.Review;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ReviewRepository extends JpaRepository<Review, Long> {
+
+    @Query("SELECT r FROM Review r WHERE r.recommendPlace.recommendId = :placeId")
+    Optional<List<Review>> findAllByPlaceId(@Param("placeId") Long placeId);
```

Comment:
```suggestion
List<Review>findAllByPlace(Place place);
```

Review와 Place는 릴레이션을 갖고 있어서 id가 아닌 Place 엔티티를 매개변수에 넣어도 좋을 거 같습니다

그리고 findAll은 데이터가 없을 경우, null이 아닌 빈 리스트를 반환하기 때문에 Optional이 필요하지 않아요!